src/datetime.js

```diff
@@ -1129,6 +1130,30 @@ export default class DateTime {
     return this.isValid ? possiblyCachedWeekData(this).weekday : NaN;
   }
 
+  /**
+   * Returns true if this date is on a weekend according to the locale, false otherwise
+   * @returns {boolean}
+   */
+  get isWeekend() {
+    return this.isValid && this.loc.getWeekendDays().includes(this.weekday);
+  }
+
+  /**
+   * Get the of the week according to the locale.
+   * 0 is the first day of the week and 6 is the last day of the week.
+   * If the locale assigns Monday as the first day of the week, then a date which is a Monday will return 0,
+   * @returns {number}
+   */
+  get localDayOfWeek() {
+    return this.isValid ? isoWeekdayToLocal(this.weekday, this.loc.getStartOfWeek()) : NaN;
+  }
+
+  get localeWeekNumber() {
```

Comment:
You used `locale` in the name here and `local` for the name in `localDayOfWeek`. We should be consistent. I prefer `local`